riscv/src/continuations.rs

```diff
@@ -6,9 +6,11 @@ use itertools::Itertools;
 use number::FieldElement;
 use riscv_executor::ExecutionTrace;
 
-use crate::bootloader::{
-    default_input, BYTES_PER_WORD, PAGE_SIZE_BYTES_LOG, PC_INDEX, REGISTER_NAMES,
-};
+mod memory_merkle_tree;
+
+use memory_merkle_tree::MerkleTree;
+
+use crate::bootloader::{default_input, PAGE_SIZE_BYTES_LOG, PC_INDEX, REGISTER_NAMES};
 
 fn make_query_callback<T: FieldElement>(bootloader_inputs: Vec<T>) -> impl QueryCallback<T> {
```

Comment:
The other PR will conflict with this as well I assume? Since this function is gone in that PR.